backend/src/test/java/quarano/actions/DiaryEventListenerTest.java

```diff
@@ -320,6 +340,31 @@ void testOnDiaryEntryMissing() {
 		assertThat(itemCaptor.getValue().getDescription().getCode()).isEqualTo(DescriptionCode.DIARY_ENTRY_MISSING);
 	}
 
+	@Test
+	void testNoActionItemForIndexPerson() {
```

Comment:
In line with what I mentioned above, I think we should just test that the relevant downstream `handle…(…)` methods are not invoked if the mocked repository invocation returns the specific case type for the event we pass in. That extensive mocking indicates we're too lost in the details of the execution rather than testing what's supposed to happen on the business level.